_episodes/04-higgs-search.md

```diff
@@ -478,7 +480,7 @@ Finally, we can include the MC and data in the same figure, and see if they are
 fig, ax = plt.subplots()
 fig.set_size_inches((10, 8))
 plot_data(stack_data_list_m4l, ranges[0], bins)
-h1 = plt.hist(
+h1 = ax.hist(
```

Comment:
also here and below: do we need to assign this to a variable?

Reply:
yes I will remove h1 = from the statements